index.bs

```diff
@@ -570,7 +547,9 @@ such as a number or a short password that could be entered by a user on a
 phone, a keyboard or a TV remote control.
 
 SPAKE2 is not symmetric and has two roles, Alice (A) and Bob (B).
-The client acts as Alice, the server acts as Bob.
+The client agent acts as Alice, the server acts as Bob.
+
+Issue: What do "client" and "server" mean here?
 
```

Comment:
It seems like the PSK presenter should be Alice and the PSK consumer should be Bob (since the PSK presenter sends the first message).  Should we change it to that?

Reply:
The text below allows either the PSK presenter or the PSK consumer to initiate authentication and send the first message.  So I think we need to define the Alice/Bob roles for both of those cases.